Dockerfile

```diff
@@ -2,15 +2,15 @@ FROM registry.access.redhat.com/ubi8/ubi-minimal:8.1 AS build-env
 
 RUN microdnf install -y git make golang
 
-ADD ./ /src
+COPY . /src/
 RUN cd /src && make code/compile
 RUN cd /src && echo "Build SHA1: $(git rev-parse HEAD)"
 RUN cd /src && echo "$(git rev-parse HEAD)" > /src/BUILD_INFO
 
 # final stage
 FROM registry.access.redhat.com/ubi8/ubi-minimal:8.1
 
-RUN microdnf update && microdnf clean all && rm -rf /var/cache/yum/*
```

Comment:
Why is this step removed?

Reply:
I realized this doesn't buy us anything.

In case of a CVE in UBI8, we get an updated package (with a CVE fix) via UBI8 image respin. Then, we need to respin our image to pick the fix up. We also don't rely on any upgraded package, so performing `microdnf update` doesn't help us in any sense. 

However, this has at least two pretty big drawbacks. At first it creates a new layer of RPMs, which may (depending on the amount of upgraded packages) increase the image size (moreover, the image size becomes unpredictable). It is far more beneficial to get those upgrades via a respun UBI8 image. The `microdnf update` procedure also has some (I guess very minimal, but still) impact on image integrity. We can be sure that pure UBI8 image has been verified by RHEL QE, but I wouldn't be so sure if they do verify the image with `microdnf upgrade` in it. I guess they do but without `microdnf upgrade` we take this question off the table.

So to sum it up - I think it does more harm than good. Since this is a new product release (and a tech preview), I think it's worth to verify my hypothesis and get rid of it.

Reply:
As per @hmlnarik request, re-introduced this piece.